src/views/api/TryItOut.vue

```diff
@@ -131,14 +140,12 @@ const requests: Requests = {
       {
         name: "q",
         example: "glucose",
-        tooltip:
-          "Search terms. Accepts any Elasticsearch query syntax, e.g. * for wildcard."
+        tooltip: "A list of search terms"
```

Comment:
The field q doesn't support lists, just a single search string.

Reply:
> Also applies to line 141.

?

Reply:
Just the tooltip for this field... I would suggest "A single search term."

Reply:
Updated the verbiage. Don't want to say "single search term" because that implies only a single word, when a single search could still be multiple words, like "fruit flies".

Reply:
https://github.com/biothings/mygeneset.info-website/pull/17/commits/d850945340420c0cd86b86efd22240ede48e5fcf